src/GovUk.Education.ExploreEducationStatistics.Content.Api.Tests/Services/BlobCacheServiceTests.cs

```diff
@@ -0,0 +1,446 @@
+﻿#nullable enable
```

Comment:
Seems to be some files on this PR that could be moved rather than deleted and added, e.g.

`MockCacheServiceExtensions` -> `MockBlobCacheServiceExtensions`.
`BlobStorageCacheServiceTests` -> `BlobCacheServiceTests`.

Not sure if that's a limitation of git when moving and making changes in the same commit? Would make reviewing a bit easier if it was possible though.

Reply:
Seems to be a limitation of Git I'm afraid. I've tried doing the renaming in a separate commit, but it still thinks it's an entirely new file(s) in the diff :shrug: 

I've reverted it back to the original commit structure as there isn't any benefit from the separate commit